src/config/config.rs

```diff
@@ -0,0 +1,273 @@
+use clap::Parser;
+
+#[derive(Debug, Parser)]
+#[clap(author, version, about, long_about = None)]
+/// A config
+pub struct Config {
+    #[clap(long = "role", value_name = "VALUE")]
+    /// Role: Controller, Node, AsyncFuse, Scheduler
+    pub role: String,
+    #[clap(long = "node-name", value_name = "VALUE")]
+    /// Node name
+    pub node_name: String,
+    #[clap(long = "node-ip", value_name = "VALUE")]
+    /// Node ip
+    pub node_ip: String,
+    #[clap(long = "mount-path", value_name = "VALUE")]
+    /// Set the mount point of FUSE
+    pub mount_path: String,
+    #[clap(long = "kv-addrs", value_name = "VALUE")]
+    /// Set the kv server addresses
+    pub kv_addr: String,
```

Comment:
Is it better to use fullname as field name and argument key? Say `kv-addresses`.

Reply:
I think `kv-sever-list` is a better name?

Reply:
`kv-servers` or `kv-endpoints` is Ok, I think it's better not to show the word `list` in the parameter names.

Reply:
Of course, it's just a naming issue, if you think it's a better name, just use it.